digits/model/tasks/caffe_train.py

```diff
@@ -703,7 +703,11 @@ def task_arguments(self, resources, env):
             if len(identifiers) == 1:
                 args.append('--gpu=%s' % identifiers[0])
             elif len(identifiers) > 1:
-                args.append('--gpus=%s' % ','.join(identifiers))
+                if config_value('caffe_root')['version'] < utils.parse_version('0.13.99'):
```

Comment:
Because the actual version is `0.14.0-alpha`, which is sorted **earlier** than `0.14.0`. That's correct according to most versioning schemes, but it makes it a little trickier to write this kind of check. I could just go ahead and use `0.14.0-alpha` instead of `0.13.99`. Does that seem better?